test-in-k8s.yaml

```diff
@@ -76,6 +76,9 @@ rules:
 - apiGroups: [""]
   resources: ["pods/log"]
   verbs: ["get","list","watch"]
+#- apiGroups: [""]
+#  resources: ["events"]
+#  verbs: ["watch"]
```

Comment:
Will need to revert that. Only the 2 tests changed by this PR should fail in this case.